prime-router/src/main/kotlin/Report.kt

```diff
@@ -76,7 +76,8 @@ data class ReportStreamFilterResult(
     val filterArgs: List<String>,
     val filteredTrackingElement: String,
     val filteredIndex: Int,
-    override val type: ActionLogDetailType = ActionLogDetailType.TRANSLATION
+    override val type: ActionLogDetailType = ActionLogDetailType.TRANSLATION,
+    val filterType: String? = ""
```

Comment:
+1 to @kevinhaube 's idea of using the `ReportStreamFilterType` (If it causes trouble when transforming to JSON a `toString` method can be defined on ReportStremFilterType, or an annotation from the jackson library can be used)